### PR TITLE
Added recipe for Faker

### PR DESCRIPTION
**Pull request by pmlandwehr**
Fresh clone  of repository, hopefully sorting out all of the rebase garbage in the last pull. (ref https://github.com/conda-forge/staged-recipes/pull/2484)

### CODE REVIEW

**Comment by pmlandwehr** [comment]
cc @jakirkham

File: recipes/faker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    - faker.providers.ssn.nl_NL
+    - faker.providers.ssn.pt_BR
+    - faker.providers.ssn.ru_RU
+    - faker.providers.ssn.sv_SE
+    - faker.providers.ssn.uk_UA
+    - faker.providers.ssn.zh_CN
+    - faker.providers.ssn.zh_TW
+    - faker.providers.user_agent
+    - faker.providers.user_agent.en_US
+    - faker.utils
```
Comment (jakirkham) [inline recipes/faker/meta.yaml]: This import list looks really long. Maybe we should move it to a `run_test.py` file to keep `meta.yaml` lean and mean.

**Comment by pmlandwehr** [comment]
Fine, but hitting the curl issue on AppVeyor - https://github.com/conda-forge/conda-smithy/issues/487

**Comment by jakirkham** [comment]
That issue should be fixed. Toggling to restart CIs.

**Comment by mariusvniekerk** [comment]
@jakirkham :shipit: ?

**Comment by jakirkham** [comment]
👍 :shipit: !

**Comment by jakirkham** [comment]
Thanks for the nudge.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**